include/Metabox/Posts/Tag.php

```diff
@@ -74,15 +74,14 @@ public function move( $options ) {
 			$current_tags = wp_get_post_tags( $post->ID, array( 'fields' => 'ids' ) );
 			$current_tags = array_diff( $current_tags, array( $options['old_tag'] ) );
 
-			if ( - 1 !== $options['new_tag'] ) {
-				if ( $options['overwrite'] ) {
-					// Remove old tags.
-					$current_tags = array( $options['new_tag'] );
-				} else {
-					// add to existing tags.
-					$current_tags[] = $options['new_tag'];
-				}
-			}
+            if ( $options['overwrite'] ) {
```

Comment:
Code is all good, but the alignment is wrong. Make sure you are using tabs with 4 spaces for intention.